apps/scroll-cd/src/app/app.component.ts

```diff
@@ -1,12 +1,20 @@
 import { AsyncPipe, NgIf } from '@angular/common';
-import { Component, HostListener } from '@angular/core';
-import { BehaviorSubject } from 'rxjs';
+import {
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  HostListener,
+  NgZone,
+} from '@angular/core';
+import { BehaviorSubject, distinctUntilChanged, fromEvent, tap } from 'rxjs';
+import { NgxDirtyCheckerModule } from '@code-workers.io/ngx-dirty-checker';
 
 @Component({
   standalone: true,
-  imports: [NgIf, AsyncPipe],
+  imports: [NgIf, AsyncPipe, NgxDirtyCheckerModule],
   selector: 'app-root',
   template: `
+    <ngx-dirty-checker></ngx-dirty-checker>
```

Comment:
What is this for ?

Reply:
That's just a helper component to visualize the number of dirty checks/ re-renderings of a component. 

https://github.com/code-workers-io/ngx-dirty-checker

However I forgot to commit the package

Reply:
Thanks 👍

Didn't know about that library. Can be useful in dev mode.